db/controllers/seed-tenM-NoSQL.js

```diff
@@ -0,0 +1,47 @@
+const database = require('../models/restaurant.js');
+const mongoose = require('mongoose');
+const {tenThousand, oneMillion} = require('../fake-nosql/generation.js');
+
+const cluster = require('cluster');
+const http = require('http');
+const numCPUs = require('os').cpus().length; // Andrea = 4
+
+if (cluster.isMaster) {
+  console.log(`Master ${process.pid} is running`);
+  let workerCount = 0;
+
+  // Fork workers.
+  for (let i = 0; i < numCPUs; i++) {
+    cluster.fork({ workerId : i });
+    workerCount++;
+  }
+
+  cluster.on('exit', (worker, code, signal) => {
+    workerCount--;
+    console.log(`worker ${worker.process.pid} died`);
+    if(workerCount === 0) {
+      mongoose.disconnect();
+      console.log('Disconnected at: ', Date.now());
+    }
+  });
+} else {
+   
+    const asyncTenThous = async (b) =>{
```

Comment:
what is b? Like what does it represent in your head? Could use a more descriptive variable